lib/detect-libc.js

```diff
@@ -72,14 +74,41 @@ const familyFromCommand = (out) => {
   return null;
 };
 
+const familyFromFilesystem = async () => {
+  if (cachedFilesystem !== undefined) {
+    return cachedFilesystem;
+  }
+  cachedFilesystem = null;
+  try {
+    cachedFilesystem = await readFile(LDD_PATH)
+      .then(content => content.includes(MUSL))
+      .then(isMusl => isMusl ? MUSL : GLIBC);
```

Comment:
We should check for the presence of the string literal `"GLIBC"` or `"glibc"` here to confirm, which follows the _make no assumptions about missing information_ theme.